### PR TITLE
Mark autogenerated files as such

### PR DESCRIPTION
**Pull request by kornelski**

### CODE REVIEW

**Comment by EPashkin** [comment]
Thanks notice this, but better use `try!(general::start_comments(w, &env.config));` instead.

**Comment by sdroege** [comment]
:+1: once @EPashkin's change is done :) Thanks!

**Comment by EPashkin** [comment]
@kornelski If you don't have time to fix PR, I will add replacement PR on next Monday.

**Comment by kornelski** [comment]
I've updated the PR

File: src/codegen/sys/build.rs [hunk 1]
```diff
@@ -22,10 +23,13 @@ pub fn generate(env: &Env) {
 }
 
 fn generate_build_script(w: &mut Write, env: &Env) -> Result<()> {
+    try!(general::start_comments(w, &env.config));
     try!(write!(
         w,
         "{}",
-        r##"extern crate pkg_config;
+        r##"
+
```
Comment (EPashkin) [inline src/codegen/sys/build.rs]: Please make only one empty line with `try!(writeln!(w));` instead

Comment (kornelski) [reply src/codegen/sys/build.rs]: That's weird. Why? 

(also why use try! when the crate requires Rust 1.19?)

Comment (kornelski) [reply src/codegen/sys/build.rs]: Updated PR

**Comment by EPashkin** [comment]
@kornelski Thanks, waiting for CI just in case.
After merge I will add PR in sys

**Pull request merged**